src/components/ChannelList/ChannelListTeam.js

```diff
@@ -45,19 +45,19 @@ const ChannelListTeam = ({
         <div className="str-chat__channel-list-team__header">
           <div className="str-chat__channel-list-team__header--left">
             <Avatar
-              image={client.user.image}
-              name={client.user.name || client.user.id}
+              image={client.user?.image}
```

Comment:
I think the overall problem with this is it breaks typescript safety if we start ignoring the types via ts-ignore based on assumptions of their current state. e.g. At some point in the future some new bit is added to the js client that would make the user undefined at a point they were on this screen. No type error would be caused as we ignored it under the assumption it would never be undefined at this point, but it would then cause a crash as we try to access the property image of undefined. I think the cleanest solution to the addition of multiple opt chains is something like `const { image, name, id, status } = client.user || {};`